src/__test__/handle-data.spec.js

```diff
@@ -7,7 +7,7 @@ describe('handle-data', () => {
   let colIdx;
 
   beforeEach(() => {
-    layout = generateLayout(2, 2, [1, 2, 0, 3]);
+    layout = generateLayout(2, 2, 200, [1, 2, 0, 3]);
```

Comment:
is it possible to have test cases to cover that the amount of cells  is larger than 10k?

Reply:
yes that is a good idea

Reply:
or we have two of those actually

Reply:
We could add one for columns > 10k. It will set rowsPerPage to 0, which is ok I think. We shouldn't support more than 10k columns, there is no useful use-case for that 😅